DCAF.Core.lua

```diff
@@ -5790,6 +5790,12 @@ DCAF.Tanker = {
     TrackSpeed = nil,         -- #number; knots 
     Track = nil, 
     Events = {},              -- dictionary; key = name of event (eg. 'OnFuelState'), value = event arguments 
+    MaxFuelLbs = {
+        ["KC130"] = 66139,
+        ["KC135"] = 199959,

```

Comment:
I got an error: Unknown tanker type: KC-135

Reply:
That's really weird as I did test with different tankers. Can you share the miz?